spec/composite-disposable-spec.coffee

```diff
@@ -29,3 +29,10 @@ describe "CompositeDisposable", ->
     expect(disposable1.disposed).toBe true
     expect(disposable2.disposed).toBe false
     expect(disposable3.disposed).toBe true
+
+  it "denies to add an object that does not respond to ::dispose", ->
```

Comment:
It’s probably better but eventually I went with `throws an error`, as it’s more consistent with [how specs are named in atom/atom](https://github.com/atom/atom/search?utf8=✓&q=throws&type=Code). Thanks! :bow: